sr_unix/ydbinstall.sh

```diff
@@ -519,6 +513,11 @@ if [ -z "$ydb_installdir" ] ; then
     else ydb_installdir=/usr/local/lib/fis-gtm/${ydb_version}_${gtm_install_flavor}
     fi
 fi
+# if install directory is relative then need to make it absolute before passing it to configure
+# (or else configure will create a subdirectory under $tmpdir (/tmp/.*) and install YottaDB there which is not what we want)
+if [ `echo $ydb_installdir | grep -c '^/'` -eq 0 ] ; then
```

Comment:
Will it still behave if it's relative starting with "../"?
Maybe consider always expanding the directory with something like ydb_installdir=$(cd $ydb_isntalldir; pwd)

Reply:
Yes it will still behave. I tried your suggestion but that will fail the "pwd" if the specified directory does not exist yet in this section of ydbinstall (will be created by configure script a little later). So I will make no change. Hope that is okay.

Reply:
That's perfectly fine, thanks for checking!